db/src/Pos/DB/Block/Epoch.hs

```diff
@@ -268,7 +321,7 @@ consolidateEpochBlocks fpath xs = ExceptT $ do
             (e:_, _) -> Left e
   where
     consolidate
-        :: ConsolidateM m
+        :: ConsolidateM ctx m
```

Comment:
I'm a bit concerned that the header length (12) (semi-) magically appears in the calculation of offsets. Could that be derived form the operation / description of the 'blnd' writer - as a guide to future readers / modifiers?

Reply:
Agreed

Reply:
I've added a comment:
```
                    SlotIndexLength (getSlotIndex lsi)
                        -- 12 == sizeof "blnd" + 2 * sizeof Word32
                        (fromIntegral $ 12 + BS.length blck + BS.length undo)
```